### ISSUE TITLE
bot freezing every 30 minutes

### ISSUE THREAD

**Issue by fabiosb01** [finndev/PokeBuddy#803]
pokebuddy freezing every 30 minutes, all pokestops disappear and shit, my ip and gps location is on the same place, tried everything, but all seems not work. any solution ? sorry for my bad english

**Comment by curtiola** [comment]
Not sure how to avoid it, there's probably more to it than IP and gps locations, could be having walk speed over 25 (app warns you not to drive and play at 25 and the bot may not press OK or whatever)
Honestly I don't know all the steps to prevent IP bans

**Comment by fabiosb01** [comment]
my walk speed is at 20 :(

**Comment by curtiola** [comment]
Yeah, I really don't know how they detect bots and distribute IP bans, could be altitude, could be device info, could be so many different things, it's hard to say, I don't know how to prevent it yet

**Comment by fabiosb01** [comment]
ok then, so much thanks

**Comment by curtiola** [comment]
Np, someone will probably figure it out within a couple days, I hope lol
